test/dummy/app/models/board.rb

```diff
@@ -0,0 +1,5 @@
+class Board < ApplicationRecord
+  has_many :tasks
+
+  broadcasts_refreshes
+end
```

Comment:
Why is a new model necessary to test this?

Reply:
I added one that actually uses the `broadcasts_refreshes` method. I'm not a Rails dev, so pls let me know if there's another way ;)

Reply:
I think adding it to an existing model would work just as well.